draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -9331,6 +9331,11 @@ Sun Nov  6 08:49:37 1994         ; ANSI C's asctime() format
    &MUST; mark the entity-tag as weak by prefixing its opaque value
    with "W/" (case-sensitive).
 </t>
+<t>
+   The Etag field can appear in the trailer section as well (see
+   (<xref target="trailer.fields"/>). Note however that recipients might
```

Comment:
seems like a comma or two is missing

Reply:
I do see an extra "(" that needs to go, but where do you miss a comma?

Reply:
Note, however, that...